llvm/test/CodeGen/AArch64/fpenv.ll

```diff
@@ -62,4 +62,47 @@ define void @func_set_rounding_downward() {
   ret void
 }
 
+define i64 @get_fpenv_01() nounwind {
```

Comment:
This is presumably not testing what you planned. The RUN line does not contain the string `globalisel`. It is testing DAG code generation. If you look at the sister file xtn.ll, it has two RUN lines to test code generation for both selectors and highlight their differences.